admin/check/check_crypto_inc.php

```diff
@@ -52,8 +52,8 @@
 
 if( config_get_global( 'login_method' ) != LDAP ) {
 	check_print_test_warn_row(
-		'login_method is set to MD5',
-		config_get_global( 'login_method' ) == MD5,
-		'MD5 password encryption is currently the strongest password storage method supported by MantisBT.'
+		'login_method is set to SAFE_HASH',
```

Comment:
Should we replace all `SAFE_HASH` to `HASH_BCRYPT`?  I suggest find across all files since there are other occurrences in other files.